views/index.pug

```diff
@@ -0,0 +1,20 @@
+extends layout.pug
+
+block head
+	header.c-header
+		.c-header-logo
+			h1 <br>Styleguide
```

Comment:
again this is all wrong

the h1 should be the logo class remove the extra div and remove the br. also the h4 below should by in the h1 and a span